lib/bigquery/dataset.js

```diff
@@ -64,6 +64,9 @@ function Dataset(bigQuery, id) {
  *     JSON array of fields, which allows for nested and repeated fields. See
  *     a [Table resource](http://goo.gl/sl8Dmg) for more detailed information.
  * @param {function} callback - The callback function.
+ * @param {?error} callback.err - An error returned while making this request
+ * @param {module:bigquery/table} callback.table - The newly created table.
```

Comment:
True, though I think at least `err` should be. I'm cool with leaving the second argument not marked as nullable. I think that would just create a lot of noise on the site.